.github/workflows/node.js.yml

```diff
@@ -2,18 +2,17 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ main ]
+    branches: [main]
   pull_request:
   workflow_dispatch:
 
 jobs:
   build:
-
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x]
+        node-version: [14.x, 16.x, 18.x, 20.x]
```

Comment:
Is there a benefit on running this on versions we dont use ourselves?

Reply:
Other people can run it on other versions, and we know it'll work when we want to switch.